x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -176,6 +176,7 @@ const getSavedObjectTypes = (): { [key: string]: SavedObjectsType } => ({
         hosts: { type: 'keyword' },
         ca_sha256: { type: 'keyword', index: false },
         ca_trusted_fingerprint: { type: 'keyword', index: false },
+        service_token: { type: 'keyword', index: false },
```

Comment:
You need to add a model version with a `mappings_addition` type of change to reflect this addition, otherwise the schema will not be updated property in MKI. 

Please refer to [the SO model version documentation](https://github.com/elastic/kibana/blob/9b59e84d3e3601a12fa5ff33814393b8f7644010/packages/core/saved-objects/core-saved-objects-server/docs/model_versions.md#adding-an-indexed-field-without-default-value)

Reply:
One question, this field is not indexed, do I still need to add a new model version? the doc says `Adding an indexed field without default value`

Reply:
> this field is not indexed, do I still need to add a new model version?

I did check, and (unless I'm wrong) your mappings for this type aren't `dynamic: false` (meaning its `strict`), so without this field being added in the mappings, trying to insert documents with it will cause an strict_mapping_thingy error

Reply:
Okay, added the new model version [here](https://github.com/elastic/kibana/pull/169252/commits/0315d0129aecfb22e55066df8a708310b8d754b5)

I noticed that the model version was not updated in a recent change in mappings [here](https://github.com/elastic/kibana/pull/169221/files#diff-4e164e3802d5171bf96a2cf9c91c20e97c5e0b74b2f93187a072d9a3139f1c18), is there any reason why it was not needed there or did we miss it?

Reply:
>  is there any reason why it was not needed there or did we miss it?

I think we just missed it... TBH we (Core) just realized last week that we overlooked that requirement in our last SO changes reviews, because security solution reported a problem about a project not having its mapping updated with their latest change in production.

Is [#169221](https://github.com/elastic/kibana/pull/169221/files#diff-4e164e3802d5171bf96a2cf9c91c20e97c5e0b74b2f93187a072d9a3139f1c18) already deployed in any MKI env / was it testedfrom a project that was already running before the promotion of the version shipping this change? My gut feeling it that it will just broke. 

(but if so, then the MV you're adding in this PR will indirectly fix it given the whole mappings will be refreshed)